test/test_endtoend.py

```diff
@@ -632,7 +632,16 @@ def setUp(self):
 
     def _check_example_echo_client_result(self, expected, stdoutdata,
                                           stderrdata):
+        # This is not right in Python 3 if the locale does not use
+        # utf-8, which is why the tests only use ASCII characters.
```

Comment:
Ditto

Reply:
I modified it to use the locale encoding here too.